Site: Add instructions for managing Calcite repos through GitHub

zabetak opened the pull request.

michaelmior commented on site/_docs/howto.md:
```diff
@@ -474,6 +474,32 @@ $ ./src/main/scripts/generate-protobuf.sh
+[ASF git repositories](https://gitbox.apache.org/repos/asf#calcite) hosting
+the source code of the project as well as the website.
+
+All repositories present on gitbox are available on GitHub with write-access 
```
gitbox -> Gitbox

michaelmior commented on site/_docs/howto.md:
accounts (ASF, GitHub) -> ASF and GitHub accounts

michaelmior commented on site/_docs/howto.md:
Set your GitHub ID into your [Apache ID profile](https://id.apache.org/)
vv
Set your GitHub username in your [Apache profile](https://id.apache.org/)

michaelmior commented on site/_docs/howto.md:
(HTTPS vs. SSH) that you are using. ->  that you are using (HTTPS vs. SSH).

michaelmior commented on site/_docs/howto.md:
"30  minutes" has an extra space

michaelmior commented on site/_docs/howto.md:
Accept invitation -> Accept the invitation

michaelmior commented:
Forgive me for being rather pedantic. This looks great. Just a few small suggested changes.

zabetak commented:
> Forgive me for being rather pedantic. This looks great. Just a few small suggested changes.

Thanks for the second pair of eyes @michaelmior and @chunweilei , much appreciated. I addressed the comments, if nothing comes up in the following days I will push it through ;)
